src/components/Header/Header.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { useHistory } from 'react-router';
+import axios from 'axios';
+import UserContext from '../../Context/UserContext';
+import FilterableList from '../FilterableList/FilterableList';
+import Button from '../Button/Button';
+import MenuList from '../MenuList/MenuList';
+import styles from './Header.scss';
+
+
+
+function Header() {
+  const [isVisible, setIsVisible] = useState(false);
+  const [cities, setCities] = useState([]);
+  const [inputValue, setInputValue] = useState('');
+  const history = useHistory();
+
+  const changeFilterableListProperties = () => {
```

Comment:
The name is rather confusing.. .what properties of filterable list do I change? It seems like you are just hiding and revealing your list instead. 

As I'm describing down below... choose one thing to hide your data. It should be enough to either set visible flag or cities. The first one seems more preferrable.